### ISSUE TITLE
google analytics for studio

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmithdemos/html5rocks#168]
```
google analytics for studio
```

Original issue reported on code.google.com by `paulir...@google.com` on 17 Aug 2010 at 10:05

**Comment by GoogleCodeExporter** [comment]
Original comment by `paulir...@google.com` on 17 Aug 2010 at 10:05
- Added labels: **Milestone-3**

**Comment by GoogleCodeExporter** [comment]
```
closed by bca5a9deb0
```

Original comment by `paulir...@google.com` on 24 Aug 2010 at 9:22
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
closed by rbca5a9deb0
```

Original comment by `paulir...@google.com` on 24 Aug 2010 at 9:22
